scripts/notifyAdmin.cfg

```diff
@@ -0,0 +1,15 @@
+/*  This configuration file is a place for your private information used to send mail.  It is expected
+to be located at /Users/Shared/notifyAdmin.cfg.  Replace the address and password in the text below to use.
```

Comment:
It looks like `/Users/Shared/` is intended to be files shared across multiple users on the machine. That introduces a risk since the gmail password is saved to the file. 

Alternatively, you can have the user set the gmail account configuration (address, password) as environment variables. The other configuration can remain in this file, and the script can just access them from this file directly.

Useful links:
* [Setting an environment variable](https://devconnected.com/set-environment-variable-bash-how-to/)
* [Accessing environment variables in Node](https://www.twilio.com/blog/working-with-environment-variables-in-node-js-html)